lldb/tools/lldb-server/lldb-platform.cpp

```diff
@@ -60,6 +71,7 @@ static struct option g_long_options[] = {
     {"max-gdbserver-port", required_argument, nullptr, 'M'},
     {"socket-file", required_argument, nullptr, 'f'},
     {"server", no_argument, &g_server, 1},
+    {"fd", required_argument, nullptr, 'd'},
```

Comment:
We don't expect this option to be used by the end user (at least not on windows, where the passing algorithm is quite complex), right?

If so, then I think we should drop the short option (the field is `int`, so I believe you can just use number not corresponding to a printable char), and use something longer for the long option -- to reduce the likelyhood of someone using this by mistake. I'd go with something like `--child-platform-fd`..